chatbot_ner/setup_logs.py

```diff
@@ -0,0 +1,114 @@
+import collections
+from typing import Dict, Any, NamedTuple, List
+
+import logging
+import structlog
+import django.dispatch
+from django.http import HttpRequest
+from django_structlog.middlewares.request import get_request_header
+from django_structlog.signals import bind_extra_request_metadata
+
+
+class _LoggingKey(NamedTuple):
+    namespace: str
+    bind_key: str
+    header_key: str
+    meta_key: str
+
+
+class LoggingKeys(object):
+    # Copied attributes from django-structlog
+    REQUEST_ID = _LoggingKey(namespace='default', bind_key='request_id',
+                             header_key='x-request-id', meta_key='HTTP_X_REQUEST_ID')
+    CORRELATION_ID = _LoggingKey(namespace='default', bind_key='correlation_id',
+                                 header_key='x-correlation-id', meta_key='HTTP_X_CORRELATION_ID')
+    # Things haptik wants bound to log lines for filtering
+    BUSINESS_ID = _LoggingKey(namespace='haptik', bind_key='business_id',
+                              header_key='x-haptik-business-id', meta_key='HTTP_X_HAPTIK_BUSINESS_ID')
+    BOT_NAME = _LoggingKey(namespace='haptik', bind_key='bot_name',
+                           header_key='x-haptik-bot-name', meta_key='HTTP_X_HAPTIK_BOT_NAME')
+    MESSAGE_ID = _LoggingKey(namespace='haptik', bind_key='message_id',
+                             header_key='x-haptik-message-id', meta_key='HTTP_X_HAPTIK_MESSAGE_ID')
```

Comment:
There is something called `coll_id`, if that is there we should also log it. `coll_id`. Unless message id is already inclusive of `coll_id`, please check and confirm if this field has a underscore or not, if not- then we need to log both. Otherwise we can just log this.

Reply:
couldn't find `coll_id`  therefore keeping only `message_id` for logging

Reply:
Can you please send me a sample message_id on Slack?